server/src/metadata/metadata.js

```diff
@@ -201,35 +228,8 @@ class Metadata {
                 }
               }).catch(reject);
             });
-          });
-      return Promise.all([ groups_ready, collections_ready, indexes_ready ]);
```

Comment:
This syntax of giving them intermediate variables with meaningful names seems better than the new version